packages/test-cases/cases/bind/sanity/output/wasm-as/Module/module.ts

```diff
@@ -0,0 +1,64 @@
+import * as Types from "..";
+
+import {
+  Read,
+  ReadDecoder,
+  Write,
+  WriteSizer,
+  WriteEncoder,
+  Box,
+  BigInt,
+  BigNumber,
+  JSON,
+  Context
+} from "@polywrap/wasm-as";
+
+import {
+  deserializemoduleMethodArgs,
+  serializemoduleMethodResult,
+  deserializeobjectMethodArgs,
+  serializeobjectMethodResult,
+  deserializeoptionalEnvMethodArgs,
+  serializeoptionalEnvMethodResult,
+  deserializeifArgs,
+  serializeifResult
+} from "./serialization";
+
+import {
+  wrap_invoke_args,
+  wrap_invoke_result,
+  wrap_invoke_error,
+  wrap_load_env,
+  InvokeArgs,
+} from "@polywrap/wasm-as";
```

Comment:
These imports can be minimized. All imports that have to do with serialization & internal `wrap_...` calls can be removed from what I can tell.

Reply:
Yeah definitely, nice catch! forget to remove it after refactor